Sources/NetworkService/InternalError/NetworkServiceInternalError+LocalizedDescription.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension NetworkServiceInternalError {
```

Comment:
You add conformance to `Error` protocol in the declaration file of type. Why do you add this property that related with  the `Error` protocol in the extension?

I think it's more properly to add `Error` protocol in the extension and add `localizedDescription` property, or to declare `localizedDescription` property in the declaration file.
WDYT?

Reply:
Conformance to an Error protocol does not require to implementation of the `localizedDescription`. If we don't provide the `localizedDescription` NetworkServiceInternalError still will conform to the `Error` protocol.

If I remove conformance from the initial declaration file, it can be confusing, when you open the implementation and don't see that `NetworkServiceInternalError` conforms to the Error protocol.

Ok, I moved the description to the initial declaration for now, as we have only one case.